pkg/kubelet/cm/node_container_manager_linux_test.go

```diff
@@ -400,3 +401,51 @@ func getEphemeralStorageResourceList(storage string) v1.ResourceList {
 	}
 	return res
 }
+
+func TestUpdateNodeAllocatableCgroups(t *testing.T) {
+	tests := []struct {
+		name             string
+		internalCapacity v1.ResourceList
+		updatedList      v1.ResourceList
+		expectedChange   bool
+	}{
+		{
+			name:             "Empty capacity",
+			internalCapacity: v1.ResourceList{},
+			updatedList:      v1.ResourceList{},
+		},
+		{
+			name:             "irrelevant capacity update",
+			internalCapacity: v1.ResourceList{},
+			updatedList:      getResourceList("10", "10Gi"),
```

Comment:
Should we add a tests testing the opposite case?

```go
internalCapacity:      getResourceList("10", "10Gi"),
updatedList:           v1.ResourceList{},
```

Reply:
No, if we want to test irrelevant updates (updates of resources that are not related to internal capacity).